### PR TITLE
Pubnet for beta

### PR DESCRIPTION
**Pull request by piyalbasu**
https://app.asana.com/0/1168666457741233/1193842482204548

We can pass a `production` flag to `extension:build` to switch to using Pubnet.
This means we  won't automatically fund a new account using friendbot and we'll check pubnet for account balance

### CODE REVIEW

File: @shared/api/internal.ts [hunk 1]
```diff
@@ -136,6 +136,7 @@ export const getAccountBalance = async (
     response = await server.loadAccount(publicKey);
   } catch (e) {
     console.error(e);
+    return { balance: "0" };
```
Comment (piyalbasu) [inline @shared/api/internal.ts:139]: this error doesn't appear to return a status code, so we can't explicitly check for 404 (which likely means it's a pubnet acct that hasn't been funded). Let's fail gracefully for now, but we can polish this experience a bit in the future

File: @shared/constants/stellar.ts [hunk 2]
```diff
@@ -1 +1,9 @@
-export const SERVER_URL = "https://horizon-testnet.stellar.org";
+declare const PRODUCTION: boolean;
+
+export const isTestnet = !PRODUCTION;
```
Comment (piyalbasu) [inline @shared/constants/stellar.ts:3]: `isTestnet` because we're adding steps in the flow based on this flag

File: extension/package.json [hunk 3]
```diff
@@ -4,8 +4,9 @@
   "license": "Apache-2.0",
   "prettier": "@stellar/prettier-config",
   "scripts": {
-    "build": "webpack --config webpack.prod.js",
+    "build": "webpack --config webpack.extension.js",
```
Comment (piyalbasu) [inline extension/package.json:7]: changing this to `extension` as build types are no longer a dev/prod binary

File: @shared/api/internal.ts (hunk 1 above)
Comment (jeesunikim) [reply @shared/api/internal.ts:139]: Can we add TODO?

**Review by jeesunikim — APPROVED** [review]

**Pull request merged**